graphene_django/tests/test_fields.py

```diff
@@ -12,17 +12,18 @@
     Article as ArticleModel,
     Film as FilmModel,
     FilmDetails as FilmDetailsModel,
+    Person as PersonModel,
     Reporter as ReporterModel,
 )
 
 
 class TestDjangoListField:
     def test_only_django_object_types(self):
-        class TestType(ObjectType):
-            foo = String()
+        class Query(ObjectType):
+            something = DjangoListField(String)
 
-        with pytest.raises(AssertionError):
-            DjangoListField(TestType)
+        with pytest.raises(TypeError):
```

Comment:
It looks like in code, this failure condition is still caused by an `assert` statement. Just curious then why this is manifesting as a TypeError now.

Reply:
Good question! The `AssertionError` is now caught and re-raise by `graphql-core` here: https://github.com/graphql-python/graphql-core/blob/0c93b8452eed38d4f800c7e71cf6f3f3758cd1c6/src/graphql/type/definition.py#L734

Reply:
Nice, thanks! Maybe it's worth adding a final line to the test to validate that the assertion message (about only using DjangoObjectTypes) is present in the raised exception string, since it seems it will be and that would make for a more thorough test

Reply:
That's a good point, done in https://github.com/graphql-python/graphene-django/pull/1492/commits/cc45b0bcdbde8b8502c6da0927e853c10bc83830